index.js

```diff
@@ -12,3 +12,26 @@ I need this code, just don't know where, perhaps should make some middleware, do
 
 Go code!
 */
+
+const express = require('express');
+const router = express.Router(); 
```

Comment:
unnecessary declaration of router, you are not using it here.

Reply:
okay, I thought I had to declare at each point. Glad to know I dont.